README.md

```diff
@@ -13,3 +13,24 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+Part 1:
+1. Depending on the data structure the worst case of one might be the best case of another.
```

Comment:
But this is what the analysis tells us. Don't understand why this is misleading.